mteb/evaluation/evaluators/RerankingEvaluator.py

```diff
@@ -202,30 +221,89 @@ def _encode_unique_texts(self, all_texts, encode_queries_func):
         )
         return all_unique_texts_embs[all_texts_indexes]
 
-    def _compute_metrics_instance(self, query_emb, docs_emb, is_relevant):
-        """Computes metrics for a single instance = (query, positives, negatives)
+    def _compute_sim_scores_instance(
+        self, query_emb: torch.Tensor, docs_emb: torch.Tensor
+    ) -> torch.Tensor:
+        """Computes similarity scores for a single instance = (query, positives, negatives)
 
         Args:
             query_emb (`torch.Tensor` of shape `(num_queries, hidden_size)`): Query embedding
                 if `num_queries` > 0: we take the closest document to any of the queries
             docs_emb (`torch.Tensor` of shape `(num_pos+num_neg, hidden_size)`): Candidates documents embeddings
+
+        Returns:
+            sim_scores (`torch.Tensor of shape `(num_pos+num_neg,)`): Query-documents similarity scores
+        """
+        sim_scores = self.similarity_fct(query_emb, docs_emb)
+        if len(sim_scores.shape) > 1:
+            sim_scores = torch.amax(sim_scores, dim=0)
+
+        return sim_scores
+
+    def _compute_metrics_instance(
+        self, sim_scores: torch.Tensor, is_relevant: List[bool]
+    ) -> Dict[str, float]:
+        """Computes metrics for a single instance = (query, positives, negatives)
+
+        Args:
+            sim_scores (`torch.Tensor of shape `(num_pos+num_neg,)`): Query-documents similarity scores
```

Comment:
```suggestion
            sim_scores: Matrix of Query-documents similarity scores of  shape `(num_pos+num_neg,)
```
general correction

Reply:
Should be fine now, just pushed the updated docstrings!